src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -919,59 +919,6 @@ public String getMfaToken() {
     return mfaToken;
   }
 
-  /**
-   * Sets the current objects if the session is not up to date. It can happen if the session is
-   * created by the id token, which doesn't carry the current objects.
-   *
-   * @param loginInput The login input to use for this session
-   * @param loginOutput The login output to ose for this session
-   */
-  void setCurrentObjects(SFLoginInput loginInput, SFLoginOutput loginOutput) {
```

Comment:
Never used and not relevant anymore. Package-private, no other applications are using this